xds/internal/balancer/ringhash/ring.go

```diff
@@ -122,24 +122,21 @@ func newRing(subConns *resolver.AddressMap, minRingSize, maxRingSize uint64, log
 // Must be called with a non-empty subConns map.
 func normalizeWeights(subConns *resolver.AddressMap) ([]subConnWithWeight, float64) {
```

Comment:
While you are here, would you mind updating the function comment to also document the second return value (and what it is used for). Thanks.

Reply:
Done